### PR TITLE
Logout function

### PR DESCRIPTION
**Pull request by lukaszmaterla**

### CODE REVIEW

File: frontend/src/components/NavigationItems.vue [hunk 1]
```diff
@@ -12,32 +12,31 @@
...
 
 export default {
   name: 'NavigationItems',
   computed: {
     ...mapState(['user']),
   },
+  methods: {
+    ...mapActions([actions.LOGOUT]),
+    logout() {
+      this[actions.LOGOUT]();
```
Comment (adamgajzlerowicz) [inline frontend/src/components/NavigationItems.vue]: this seems to be here twice now

File: frontend/src/components/Status.vue [hunk 2]
```diff
@@ -14,6 +18,7 @@ export default {
   name: 'Refresh',
   computed: {
     ...mapState([IS_ONLINE]),
+    ...mapState(['user']),
```
Comment (adamgajzlerowicz) [inline frontend/src/components/Status.vue]: I believe that those two lines can be joined into one.

**Comment by adamgajzlerowicz** [comment]
Can we actually add the `/logout` path to the router (`src/router/index.js`)?  Currently visiting /logout doesn't trigger the logout action, but redirects to `/`

**Comment by lukaszmaterla** [comment]
I'm not sure, but this is because of beforeEach method in index.js.
Do we actually need this route? I tried to change this, and execute logout actions from store but it didn't work in beforeEnter hook.

**Comment by adamgajzlerowicz** [comment]
For this a new route is required in `src/router/index.js`. This route will have a component that will be dedicated to perform a logout and redirect back to login after the logout. I hope that this is making sense. PM if it doesn't 😄

**Comment by jacol12345** [comment]
Merged as part of #138

**Pull request closed without merging**